archunit-junit/junit5/engine/build.gradle

```diff
@@ -62,7 +68,3 @@ def configureDependencies = { deps ->
     }
 }
 this.with project(':archunit-junit').configureJUnitArchive(configureDependencies)
-
-singlePackageExport {
-    exportedPackage = 'com.tngtech.archunit.junit.internal'
```

Comment:
@codecholeric Could you please suggest what I should do about this one?

If the other PRs are merged, there's going to be a lot more filtering stuff, so I kind of felt it would be clearer if those resided in a separate subpackage...